src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,110 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    public static final int ZERO = 0;
+    private static final int DEFAULT_INITIAL_CAPACITY = 1 << 4;
+    private static final float DEFAULT_LOAD_FACTOR = 0.75f;
+    private Node<K, V>[] table;
+    private int size;
+    private int threshold;
+    private int currentCapacity;
+
+    public MyHashMap() {
+        table = new Node[DEFAULT_INITIAL_CAPACITY];
+        currentCapacity = table.length;
+        threshold = (int) (currentCapacity * DEFAULT_LOAD_FACTOR);
+    }
 
     @Override
     public void put(K key, V value) {
-
+        resize();
+        Node<K, V> newNode = key != null ? new Node<>(key.hashCode(), key, value, null) :
```

Comment:
create constuctor for `Node` only with two parameters : `key`, `value`. the `Node.hashCode` is redundant field. 
and you don't need pass the `null` value for the `Node.next`